scripts/wakeword/gen_topo.pl

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env perl
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)
+
+# Generate a topology file.  This allows control of the number of states in the
```

Comment:
can you please modify the copyright header and clarify what the differences are from the standard gen_topo.pl script?  Have a look and see if any other files have these types of issues.  In general for all copied+modified code, e.g. the decoder code, you should make clear what it was modified from and how and why it differs.

Reply:
I just pushed a change to copyright headers and clarification for some code/scripts which are modified from other existing ones.